### PR TITLE
Siamcat2 interpretation plot

### PR DESCRIPTION
**Pull request by Ales-ibt**
The notebook is now complete, please give it a go and let me know if you consider it necessary to make any changes.

### CODE REVIEW

File: dependencies/populate-mgnifyr-cache.R [hunk 1]
```diff
@@ -3,7 +3,8 @@ library(MGnifyR)
...
 metadata = mgnify_get_analyses_metadata(mg, tara_all)
-
+clean_acc=c('MGYA00590519','MGYA00590489','MGYA00590465','MGYA00590530','MGYA00590487','MGYA00590476','MGYA00590516','MGYA00590561','MGYA00590536','MGYA00590567','MGYA00589017','MGYA00590491','MGYA00590493','MGYA00593140','MGYA00590557','MGYA00589029','MGYA00589036','MGYA00593111','MGYA00593018','MGYA00593225','MGYA00590531','MGYA00589062','MGYA00590459','MGYA00589052','MGYA00593131','MGYA00589048','MGYA00590527','MGYA00590576','MGYA00589058','MGYA00590558','MGYA00590550','MGYA00593113','MGYA00593015','MGYA00593121','MGYA00590572','MGYA00590545','MGYA00590518','MGYA00593126','MGYA00590526','MGYA00590500','MGYA00590570','MGYA00590520','MGYA00590443','MGYA00589013','MGYA00590449','MGYA00589021','MGYA00593130','MGYA00589047','MGYA00589042','MGYA00590577','MGYA00590470','MGYA00590473','MGYA00593216','MGYA00590562','MGYA00590464','MGYA00590484','MGYA00590462','MGYA00590565','MGYA00590439','MGYA00590472','MGYA00590566','MGYA00590552','MGYA00590485','MGYA00593133','MGYA00590544','MGYA00590455','MGYA00590437','MGYA00589044')
+ps = mgnify_get_analyses_phyloseq(mg, clean_acc)
```
Comment (SandyRogers) [inline dependencies/populate-mgnifyr-cache.R:10]: brilliant

File: src/notebooks/R Examples/Comparative Metagenomics.ipynb [hunk 2]
```diff
@@ -1252,7 +1285,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model.interpretation.plot(siamcat_obj, consens.thres = 0.8, fn.plot = 'interpretation_plot.pdf')"
+    "model.interpretation.plot(siamcat_obj, consens.thres = 0.5, fn.plot = 'interpretation_plot.pdf')"
```
Comment (SandyRogers) [inline src/notebooks/R Examples/Comparative Metagenomics.ipynb:1288]: So, I still got `Error in model.interpretation.plot(siamcat_obj, consens.thres = 0.5, fn.plot = "interpretation_plot.pdf"): Some models do not contain any features!` (and an empty plot) here even with these changes... were you expecting this to now work?

**Review by SandyRogers — COMMENTED** [review]
Thanks for working on this again @Ales-ibt ! I still got a missing plot and error on the final SIAMCAT step... did you get it working?

File: dependencies/populate-mgnifyr-cache.R (hunk 1 above)
Comment (Ales-ibt) [reply dependencies/populate-mgnifyr-cache.R:10]: I'll update this again, as I change the data once more

**Review by SandyRogers — APPROVED** [review]
Brilliant, @Ales-ibt , thank you!

**Pull request merged**